source/authentication/custom-jwt.txt

```diff
@@ -618,10 +618,10 @@ Firebase JWT Authentication
 ---------------------------
 
 You can use the `Firebase Authentication <https://firebase.google.com/docs/auth>`__
-service to authenticate users into Realm apps as well as Firebase apps. 
-Use Realm Custom JWT authentication to access Realm services with Firebase credentials. 
+service to authenticate users into App Services apps as well as Firebase apps. 
```

Comment:
It's not a dealbreaker, but "App Services app(s)" just does not sound great IMO. Wish we had a better alternative.